drivers/hmis/spec/requests/hmis/client_files_controller_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative 'login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.describe Hmis::ClientFilesController, type: :request do
+  include_context 'hmis base setup'
+  include_context 'file upload setup'
+
+  let!(:access_control) do
+    create_access_control(
+      hmis_user,
+      ds1,
+      with_permission: [
+        :can_view_clients,
+        :can_view_any_nonconfidential_client_files,
+      ],
+    )
+  end
+
+  # let!(:client) { create(:hmis_hud_client, data_source: ds1, user: u1) }
```

Comment:
Maybe we don't need this because c1 is created as part of the `hmis base setup`?